### ISSUE TITLE
Tab remapping not working?

### ISSUE THREAD

**Issue by karmaral** [vscode-neovim/vscode-neovim#1073]
Hello. I'm having the already mentioned issue of indenting a visual block only working for the first line, so I'm trying to delegate it entirely to nvim's indentation.

I have in my `init.vim`
```vim
nnoremap <tab> >>
nnoremap <s-tab> <<
vnoremap <tab> >
vnoremap <s-tab> <
```
But it doesn't seem to have any effect on VSCode.
I tried disabling all of the `tab` related shortcuts, or even sending a `<tab>` through a `vscode-neovim.send` command, but none of it works.

I've also tried the opposite route with 
```vim
if exists('g:vscode')
 nnoremap <tab> <Cmd>call VsCodeNotify('editor.action.indentLines')
 nnoremap <s-tab> <Cmd>call VsCodeNotify('editor.action.outdentLines')
 vnoremap <tab> <Cmd>call VsCodeNotifyVisual('editor.action.indentLines')
 vnoremap <s-tab> <Cmd>call VsCodeNotifyVisual('editor.action.outdentLines')
```
But that doesn't do anything either.
I apologize if this is obvious. I've searched all the issues I've found on the matter but none of their solutions seem to work.

Could some other vscode binding to the `tab` key be interfering? I would assume that they wouldn't if they don't meet the appropiate criteria.

Thanks in advance.

**Comment by craigmac** [comment]
This works for me (Not sure why NotifyVisual didn't work but switching to CallVisual works:

```vim
nnoremap <silent> <Tab> <Cmd>call VSCodeNotify('editor.action.indentLines')<CR>
nnoremap <silent> <S-Tab> <Cmd>call VSCodeNotify('editor.action.outdentLines')<CR>

xnoremap <silent> <Tab> <Cmd>call VSCodeCallVisual('editor.action.indentLines', 1)<CR>
xnoremap <silent> <S-Tab> <Cmd>call VSCodeCallVisual('editor.action.outdentLines', 1)<CR>
```

**Comment by karmaral** [comment]
Ah, right.
Wow I've missed on quite a few details 😅 I was omitting the second argument for the visual one, and also doing V`s`Code instead of V`S`Code. 

In that case yeah, `VSCodeNotifyVisual` works just fine.

Also, I tried it in an almost-bare vscodium install and everything worked as expected, but not on my vscode install.

Turns out I had specified the folder to the `init.vim` but not the file itself. I don't know what weird overlap I was having. Ended up just using the `nvim` as the path, like the default install, and let it automatically load the init file on its own.

I guess it works! Thanks for replying, I had kinda gotten used to the ugly workarounds already.
